### DISCUSSION TITLE
Edit/Update icon

### DISCUSSION THREAD

**Discussion by atulhajare** [Budibase/budibase#8902] [Help] [answered]
I have a table that has some records.
Want to edit the selected row from the icon placed outside the table.
If the edit icon button is inside the table, it is working. But as per the requirement, we need the icon outside the table.
Is it possible?
![Screenshot (43)](https://user-images.githubusercontent.com/119647906/205217543-9913c31b-d9fb-45e5-91bc-13cd52acee39.png)

**Comment by melohagan** [comment, marked as answer]
Hey @atulhajare 

This is possible! Hopefully you find this answer useful: https://github.com/Budibase/budibase/discussions/8759#discussioncomment-4207094

**Reply by atulhajare** [reply]
Thank you. @melohagan this is useful.
